dev-docs/pb7-notes.md

```diff
@@ -0,0 +1,77 @@
+
+| Module      | Replacement |
+|:-----------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AdLive Bid Adapter
+| Akamai ID Submodule | rearchitected to use RTD module
+| AppNexus Analytics Adapter
+| DistrictM DMX Bid Adapter | replaced by Sharethrough
+| FLOC ID submodule
+| Halo ID & RTD submodules | replaced by Hadron
+| NextRoll ID submodule
+| OneVideo Bid Adapter | replaced by YahooSSP
+| Sortable Adapters | replaced by Freestar
+| TrustX Bid Adapter | now an alias of Grid
+| UserId Targeting Module
+
+## Adapter Interface
+
+Following the precedent on Prebid 5, bidders should read additional values from the ad unit, global config, or the ortb2 object. Bidders may still take these values as parameters, but should also consider or support the more standard interface. Publishers do not want to set these values multiple times, one for each bid parter. Publishers can now rely on certain parameters no longer needing to be set in bidder configuration, including the instl flag on an ad unit, the position parameter, and the banned categories (bcats). Bidders should consider that if they are accepting any openrtb field in their configuration, they should also check the ortb2 object for that parameter. 
+
+In the Prebid 5 release notes, it was noted that publishers should no longer use publisherDomain as a setConfig parameter, and instead prefer PageURL. Adapters no longer read from this location. Also, the object presented to bidders with the page url and that page's referring page url is essentially redone, and our goal is that adapters can rely on ortb2.site.page and ortb2.site.ref, with flags for when top is not reached, the canoncical link is used, setConfig('pageUrl') is used, or when window.location.href is used for ortb2.site.page. Many bidders had disparate logic for this and the refererInfo.referer seemed to have different meanings in different contexts.
+
+Also, Bid Adapters (not all adapters) no longer have access to storage manager unless explicitly whitelisted by the publisher. We believe bidding functionality should rarely if ever need this access and that this extra functionality included in bid adapters must be consented to by the installer in their configuration. The following bid adapters' unit tests were affected by this change: Adagio, Adnuntius, AP Stream, Concert, Conversant, Craft, Criteo, E-Planning, Invibes, Kargo, Quantcast, Trion, Unicorn, and Vidazoo. Adapters simply setting a random identifier in the first party local storage or cookie should consider if the popular shared id user id submodule can fully achieve this functionality for them. See storageAllowed in the [Publisher API Reference]({{site.baseurl}}/dev-docs/publisher-api-reference.html) 
```

Comment:
```suggestion
Also, Bid Adapters (not all adapters) no longer have access to storage manager unless explicitly allowed by the publisher. We believe bidding functionality should rarely if ever need this access and that this extra functionality included in bid adapters must be consented to by the installer in their configuration. The following bid adapters' unit tests were affected by this change: Adagio, Adnuntius, AP Stream, Concert, Conversant, Craft, Criteo, E-Planning, Invibes, Kargo, Quantcast, Trion, Unicorn, and Vidazoo. Adapters simply setting a random identifier in the first party local storage or cookie should consider if the popular shared id user id submodule can fully achieve this functionality for them. See storageAllowed in the [Publisher API Reference]({{site.baseurl}}/dev-docs/publisher-api-reference.html) 
```

preferable _allowed_ or _included_ instead of whitelisted